src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/bail/homeoffice/email/HomeOfficeBailSignedDecisionNoticeUploadedPersonalisation.java

```diff
@@ -0,0 +1,78 @@
+
+    private final String homeOfficeBailSignedDecisionNoticeUploadedPersonalisationTemplateId;
+    private final String homeOfficeBailSignedDecisionNoticeUploadedWithoutLrPersonalisationTemplateId;
+    private final String bailHomeOfficeEmailAddress;
+
+
+    public HomeOfficeBailSignedDecisionNoticeUploadedPersonalisation(
+        @NotNull(message = "homeOfficeBailSignedDecisionNoticeUploadedPersonalisationTemplateId cannot be null")
+        @Value("${govnotify.bail.template.uploadSignedDecisionNotice.email}") String homeOfficeBailSignedDecisionNoticeUploadedPersonalisationTemplateId,
+        @Value("${govnotify.bail.template.uploadSignedDecisionNoticeWithoutLR.email}") String homeOfficeBailSignedDecisionNoticeUploadedWithoutLrPersonalisationTemplateId,
+        @Value("${bailHomeOfficeEmailAddress}") String bailHomeOfficeEmailAddress
+    ) {
+        this.homeOfficeBailSignedDecisionNoticeUploadedPersonalisationTemplateId = homeOfficeBailSignedDecisionNoticeUploadedPersonalisationTemplateId;
+        this.homeOfficeBailSignedDecisionNoticeUploadedWithoutLrPersonalisationTemplateId = homeOfficeBailSignedDecisionNoticeUploadedWithoutLrPersonalisationTemplateId;
+        this.bailHomeOfficeEmailAddress = bailHomeOfficeEmailAddress;
+    }
+
+    @Override
+    public String getReferenceId(Long caseId) {
+        return caseId + "_BAIL_UPLOADED_SIGNED_DECISION_NOTICE_HOME_OFFICE";
+    }
+
+    @Override
+    public String getTemplateId(BailCase bailCase) {
+        return isLegallyRepresented(bailCase)
+            ? homeOfficeBailSignedDecisionNoticeUploadedPersonalisationTemplateId
+            : homeOfficeBailSignedDecisionNoticeUploadedWithoutLrPersonalisationTemplateId;
+    }
+
+    @Override
+    public Set<String> getRecipientsList(BailCase bailCase) {
+        return Collections.singleton(bailHomeOfficeEmailAddress);
+    }
+
+    @Override
+    public Map<String, String> getPersonalisation(BailCase bailCase) {
+        requireNonNull(bailCase, "bailCase must not be null");
+
+        if (isLegallyRepresented(bailCase)) {
+            return ImmutableMap
+                .<String, String>builder()
+                .put("bailReferenceNumber", bailCase.read(BailCaseFieldDefinition.BAIL_REFERENCE_NUMBER, String.class).orElse(""))
+                .put("legalRepReference", bailCase.read(BailCaseFieldDefinition.LEGAL_REP_REFERENCE, String.class).orElse(""))
+                .put("applicantGivenNames", bailCase.read(BailCaseFieldDefinition.APPLICANT_GIVEN_NAMES, String.class).orElse(""))
+                .put("applicantFamilyName", bailCase.read(BailCaseFieldDefinition.APPLICANT_FAMILY_NAME, String.class).orElse(""))
+                .put("homeOfficeReferenceNumber", bailCase.read(BailCaseFieldDefinition.HOME_OFFICE_REFERENCE_NUMBER, String.class).orElse(""))
+                .put("decision", isBailGranted(bailCase) ? "Granted" : isBailRefused(bailCase) ? "Refused" : "")
```

Comment:
Why not use the check/result you have below on line 74?

Reply:
You're right, they should be the same, but I think they should both include the empty string option as 3rd option for safety.
Normally we can expect that field to always be populated with either `grant/conditionalGrant` or `refused`. But looking at [tests for other pre-existing personalisations](https://github.com/hmcts/ia-case-notifications-api/blob/34f3c378a453286479a7532d666fec37c7c27909/src/test/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/bail/applicant/ApplicantBailApplicationSubmittedPersonalisationSmsTest.java#L86), it looks like the default behavior should be that if that field is empty, then it should write an empty string. Or the same as the lines above where there's `orElse("")`.

I refactored it into a method `decision(bailCase)` that I've put as default method in the interface `BailEmailNotificationPersonalisation`, so I can always reuse the same method instead of having two (`isBailGranted()` and `isBailRefused()`)